src/tests/ftest/launch.py

```diff
@@ -1100,12 +1102,18 @@ def run_gdb(pattern):
             exe_name_start = exe_type.find("execfn: '") + 9
             exe_name_end = exe_type.find("', platform:")
             exe_name = exe_type[exe_name_start:exe_name_end]
-            get_output('cd {0} && gdb -ex "set pagination off" \
-                            -ex "thread apply all bt full"     \
-                            -ex "detach"                       \
-                            -ex "quit"                         \
-                        {1} {2} > {2}.stacktrace'.format(
-                            daos_cores_dir, exe_name, corefile))
+            cmd = [
+                "gdb", "-cd={}".format(daos_cores_dir),
+                "-ex", "\"set paginator off\"",
```

Comment:
Seems the name of this option got changed in the patch?
```suggestion
                "-ex", "\"set pagination off\"",
```

Reply:
Thanks for catching this - it's `paginator`  for avocado and `pagination` for gdb.  Fixed.